website/docs/sdk-and-devtool/lumos.mdx

```diff
@@ -0,0 +1,283 @@
+  <TabItem value="print-balance" label="Command">
+```
+
+```bash
+npx ts-node wallet.ts
+```
+
+```mdx-code-block
+  </TabItem>
+  <TabItem value="view-balance" label="Result">
+```
+
+```bash
+address: <your-address>
+balance: 0 CKB
+```
+
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
+
+It looks like we don't have any CKB yet, but don't worry, we can claim CKB from [Pudget Faucet](https://faucet.nervos.org/). After inputting your address and claiming 10000.0 CKB, wait about 10s, and run the above command to check your balance again. You should see the change in your balance.
+
+```bash
+address: <your-address>
+balance: 10000 CKB
+```
+
+### Build a Transfer Transaction
+
+Suppose Bob is a pizzeria and you have ordered a pizza from Bob for 100 CKB. Now you need to pay Bob. Bob's address is: `ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqgy5rtexzvhk7jt7gla8wlq5lztf79tjhg9fmd4f`.
+
+Update the `wallet.ts` to build the transfer:
+
+```ts title='wallet.ts'
+import { commons, Address, HexString } from "@ckb-lumos/lumos";
+
+const transfer = async (
+  from: Address,
+  to: Address,
+  capacity: number,
+  privateKey: HexString
+) => {
+  let txSkeleton = helpers.TransactionSkeleton({ cellProvider: indexer });
+
+  // Build the base transaction
+  txSkeleton = await commons.common.transfer(
+    txSkeleton,
+    [from],
+    to,
+    BigInt(capacity)
+  );
+  // Pay the transaction fee
+  txSkeleton = await commons.common.payFeeByFeeRate(txSkeleton, [from], 1000);
+
+  // Prepare signing entried and sign the transaction
+  txSkeleton = commons.common.prepareSigningEntries(txSkeleton);
+  const message = txSkeleton.get("signingEntries").get(0)?.message;
+  const Sig = hd.key.signRecoverable(message!, privateKey);
+  const tx = helpers.sealTransaction(txSkeleton, [Sig]);
+
+  // Send the transaction
+  return rpc.sendTransaction(tx, "passthrough");
+};
+
+const bobAddress =
+  "ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqgy5rtexzvhk7jt7gla8wlq5lztf79tjhg9fmd4f";
+transfer(address, bobAddress, 100 * 10 ** 8, privateKey).then(
+  (txHash: string) => console.log("txHash: ", txHash)
+);
+```
+
+Run the following command to execute the transfer:
+
+```mdx-code-block
+<Tabs>
+  <TabItem value="build-transfer" label="Command">
+```
+
+```bash
+npx ts-node wallet.ts
+```
+
+```mdx-code-block
+  </TabItem>
+  <TabItem value="view-transfer" label="Result">
+```
+
+```bash
+txHash is: 0x998dd1ec986ad0f1aff5f527dc3e9fe13cba0d20ee2e7ee0ce83af213cd399c4
+```
+
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
+
+You can look up the transaction by pasting your `txHash` to the [Explorer](https://pudge.explorer.nervos.org/). This transaction consumed a 10000 CKB Cell and generated a 100 CKB Cell for Bob as well as generated change for itself.
+
+---
+
+## Additional Resources
+
+- [Github](https://github.com/ckb-js/lumos/)
+- [Examples](https://github.com/ckb-js/lumos/tree/develop/examples)
+- [Lumos Docs](https://lumos-website.vercel.app/)
```

Comment:
can we add a link to dapp tutorial here? since the dapp part is heaviely using lumos. the title can be something like `DApp Tutorial`